README.md

```diff
@@ -1,72 +1,135 @@
-# dsaps
+# DSpace API Python Scripts
 
-This command line application provides several ways of interacting with the [DSpace](https://github.com/DSpace/DSpace) API. This application was written for DSpace 6.3, it has not been tested against other DSpace versions. Previously, this branch of the repository was a set of self-contained scripts that could be run independently, those scripts can be found as a [release](https://github.com/MITLibraries/dspace-api-python-scripts/releases/tag/v1.0).
+DSpace API Python Scripts (DSAPS) is a Python CLI application for managing uploads to DSpace. DSAPS has only been used on instances running DSpace 6.3 and has not been tested on other versions.
+
+Note: Previously, the repository comprised of self-contained scripts that could be run independently. Those scripts can be found as a [release](https://github.com/MITLibraries/dspace-api-python-scripts/releases/tag/v1.0).
+
+## Development
+
+- To preview a list of available Makefile commands: `make help`
+- To install with dev dependencies: `make install`
+- To update dependencies: `make update`
+- To run unit tests: `make test`
+- To lint the repo: `make lint`
+- To run the app: `pipenv run dsaps --help`
+
+The example below shows how to run the `reconcile` command:
 
-## Installation
-Clone the repository and install using [pipenv](https://github.com/pypa/pipenv):
-```
-pipenv install
-```
-After installation, run the application with:
 ```
-pipenv run dsaps
+pipenv run dsaps --url $DSPACE_URL -e $DSPACE_EMAIL -p $DSPACE_PASSWORD reconcile -m <metadata-csv> -o /output -d <content-directory> -t <file-type>
```

Comment:
This feels out of place, could we instead do examples below for all 3 commands after they are defined?

Reply:
@ehanson8 Let me know what you think of the change introduced in the latest commit!